testcontainers/src/test/java/io/debezium/examples/testcontainers/DebeziumContainerTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.testcontainers;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.rnorth.ducttape.unreliables.Unreliables;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+
+import com.jayway.jsonpath.JsonPath;
+
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
+import io.debezium.testing.testcontainers.DebeziumContainer;
+
+public class DebeziumContainerTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DebeziumContainerTest.class);
+
+    private static Network network = Network.newNetwork();
+
+    private static KafkaContainer kafkaContainer = new KafkaContainer()
+            .withNetwork(network);
+
+    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>("debezium/postgres:11")
```

Comment:
Could this be also exteranlized as system property?

Reply:
Sure, but is it worth it? I'd like to keep things simple.